plugins/templates/README.md

```diff
@@ -27,12 +27,12 @@ server.register({
 
 ### Routes
 
-#### Get all templates
-`page` and `count` optional
+#### Template
+##### Get all templates
```

Comment:
Hm this heading style gets interesting because it looks smaller than the *Arguments*